Meetings/2021-11-01-meeting-minutes.md

```diff
@@ -0,0 +1,91 @@
+
+Our charter should be our overall goals for this working group, not individual proposals, or changes that we would like to see implemented.
+
+The projects committee handles approval of projects joining the foundation, onboarding support and is looking at ways to support smaller projects that are not part of the foundation yet, seed projects. We don't want to overlap with their mandate so should stick with support for projects that are onboarded into the foundation.
+
+### Charter Discussion
+
+- Provide a forum for existing project maintainers to provide feedback to the board on support that they need or desire
+- A group of people that the board can bring policies to get their feedback and desires, a sounding board for the board
+- Will work with onboarded projects
+- Helping the board help projects and helping maintainers help each other
+- Committee is open to everyone in the community
+- Will rework the charter and add to GitHub issue for further discussion
+
+### Questions for the board
+
+- Where do we stand with projects that are no longer maintained? Do we archive or help move them forward?
+- What is the process for someone else to take over a project?
+- What are requirements vs recommendations for member projects?
+- How do we deal with new requirements for existing projects? Can they be retroactively applied?
+
+### Future Topics for Discussion
+
+- We should document and help with best practices for member projects
+- Committees are driven by the committee members
+- Should be a way for the community to ask for a committee to work on a project
+- Issue template for proposals
+- How does the community provide feedback, propose new policies and best practices?
+- We need a consolidated database of project maintainers that we can reference for help between projects
+- Create an area owners doc for project maintainership across the foundation. This would list skillsets of maintainers so that we can reach out to and help each other
+- A chance for maintainers to interact and share information
+- [Further ideas linked by Tanner](https://github.com/dotnet-foundation/Home/discussions/52#discussioncomment-1453758)
+- Should not make decisions on behalf of all projects, should only provide feedback
+- How do we communicate with all maintainers?
+- How do we find reliable contacts for all projects?
+- Maintainers need to feel that their thoughts and opinions are being heard from
+- A go/no-go poll of maintainers for new project requirements and recommendations
+- Documenting and educating maintainers on why policies are important
+- [Library Guidance](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/) was created by James. We could use as a basis of our best practices and expand?
+- Project committee does not onboard a project, that is a gap in support
```

Comment:
Was this meant to be:

```suggestion
- Project committee only onboards a project, there is a gap in support
```

?

Reply:
That was intentional and surprised me too. One of the members of the project committee said that they recommend projects but are not currently helping onboard them!

Reply:
Oh, interesting! Perhaps to clarify then:
```suggestion
- Project committee does not onboard a project, it only recommends which ones should be onboarded. That is a gap in support.
```